cli-openvm/src/main.rs

```diff
@@ -261,3 +297,33 @@ pub fn run_with_metric_collection_to_file<R>(file: std::fs::File, f: impl FnOnce
         .unwrap();
     res
 }
+
+fn maybe_compute_empirical_constraints(
+    guest_program: &OriginalCompiledProgram,
+    powdr_config: &PowdrConfig,
+    stdin: StdIn,
+) {
+    if !powdr_config.optimistic_precompiles {
+        return;
+    }
+
+    tracing::warn!(
+        "Optimistic precompiles are not implemented yet. Computing empirical constraints..."
```

Comment:
Is this still relevant?

Reply:
The warning? I think so, this PR doesn't add optimistic precompiles, right?